websocket/config.go

```diff
@@ -0,0 +1,42 @@
+package polygonws
+
+import "time"
+
+// todo: add more config options with validation (waits, feed/market, etc)
```

Comment:
I forgot to mention this the other day, but I wanna suggest we support a config option that doesn't parse messages (except for maybe status messages). Something like this in the jvm client: https://github.com/polygon-io/client-jvm/blob/master/src/main/kotlin/io/polygon/kotlin/sdk/websocket/PolygonWebSocketClient.kt#L61-L64

My main motivation for including that in the jvm client was so if for whatever reason the models in this client get out of sync with what the websocket actually sends, users will have the ability to parse messages using their own models.

Not saying add that to this PR now or anything, just wanted to share a thought

Reply:
i like it! left a todo comment about it. i'll figure out a good way to handle it as we get subscriptions and data handling in place